xsd/NeTEx_publication.xsd

```diff
@@ -285,12 +285,17 @@ Correct COnstraints for PointOnRoute
+			<xsd:field xpath="@dataSourceRef"/>
+			<!-- <xsd:field xpath="@version"/>  We don't check the version for the dataSourceRef any longer. +v2.0 -->
 		</xsd:keyref>
 		<xsd:key name="DataSource_AnyVersionedKey">
 			<xsd:selector xpath=".//netex:DataSource"/>
 			<xsd:field xpath="@id"/>
-			<xsd:field xpath="@version"/>
+			<!-- <xsd:field xpath="@version"/> +v2.0 -->
```

Comment:
I think you should really remove it. Because from the perspective of the object refering to a DataSource it certainly will not be the same version.